src/pages/contributors/tim-jacomb.adoc

```diff
@@ -0,0 +1,67 @@
+= Tim Jacomb
+:page-name: Tim Jacomb
+:page-linkedin: 
+:page-twitter: tjaynz
+:page-github: timja
+:page-email: 
+:page-image: avatar/tim-jacomb.jpg
+:page-pronouns: He/Him/His
+:page-location: Brighton, England
+:page-firstcommit: 2014
+:page-datepublished: 2024-02-21
+:page-featured: true
+:page-intro: Tim Jacomb is a software engineer based out of Brighton, England and has been involved with the Jenkins project for over 10 years. Over this time, he has implemented several UI/UX updates & optimizations, reviewed thousands of pull requests for multiple areas of Jenkins, and currently serves as the Release officer.
+
+Tim is originally from New Zealand and has a passion for exploration.
```

Comment:
It might just be me (it probably is), but I wince every time I read that someone has a passion for something.

Maybe he is an explorer at heart? Or his first love is exploration? They're a little cliched too, but not so much as 'passion' :)

Reply:
Definitely get that, thanks @liamfield, I've changed the wording a bit to be a bit less cliché